.github/workflows/main.yml

```diff
@@ -25,9 +25,6 @@ jobs:
       - name: Lint
         run: yarn lint
 
-      - name: Test
-        run: yarn test --ci --coverage --maxWorkers=2
```

Comment:
Ooh are we removing this because we didn't have any tests?

Reply:
yes :) lolol